Composer/packages/server/src/locales/en-US.json

```diff
@@ -245,6 +245,9 @@
   "are_you_sure_you_want_to_uninstall_these_extension_cf6265e8": {
     "message": "Are you sure you want to uninstall these extensions?"
   },
+  "arm_template_bc8441bb": {
```

Comment:
was this file intentionally comitted?

Reply:
Hmm this seems to be adding the strings from my previous PR. @beyackle what is the desired process for this. I imagine any PR that adds new localized strings should run a build and also check in the autogenerated locale file. Is that the case?
If so then I should have checked in these locale files with the PR that created these autogenerated changes. If thats the case I will bundle these changes in with the PR and ensure future localized strings are checked in with their respective PR.

Reply:
Ben has confirmed that that is the case